stellar-contract-env-host/Cargo.toml

```diff
@@ -13,6 +13,7 @@ num-bigint = "0.4"
 num-rational = "0.4"
 thiserror = "1.0.31"
 wasmi = { version = "0.11.0", optional = true }
+rstest = "0.13.0"
```

Comment:
Can you add this to the `dev-dependencies`? It should not be required outside tests.

Reply:
Yes, done.